packages/teams-js/src/public/dialog.ts

```diff
@@ -102,6 +102,8 @@ export namespace dialog {
    *
    * @param result - The result to be sent to the bot or the app. Typically a JSON object or a serialized version of it
    * @param appIds - Valid application(s) that can receive the result of the submitted dialogs. Specifying this parameter helps prevent malicious apps from retrieving the dialog result. Multiple app IDs can be specified because a web app from a single underlying domain can power multiple apps across different environments and branding schemes.
+   *
+   * MPORTANT -  When in a ContentActions scenario, the passed in result will not go anywhere as there is no  registered "parent" to send the result to
```

Comment:
I might remove the "IMPORTANT" and add this to the description on the result parameter.

2 other thoughts:

1) It would be good to be more specific about what a "ContentActions" scenario is. Possibly by linking to the part of the context that would be set?
2) Is the value of submit always ignored? I don't remember content actions supporting bot ids, but if they *do* support opening dialogs with completionBotIds then we should make sure to be specific about that as well. May be a good thing to confirm with PM if we don't already know.

Reply:
thank you for the feedback! after following up, I will follow up with PM about the bot scenario and update the comment as needed